app/build.gradle.kts

```diff
@@ -57,6 +57,9 @@ dependencies {
     debugImplementation(libs.androidx.ui.tooling)
     debugImplementation(libs.androidx.ui.test.manifest)
 
+    // Icon Extended
+    implementation("androidx.compose.material:material-icons-extended:1.7.8")
```

Comment:
Please use the `libs.versions.toml` for managing dependencies, please also move the Sora Editor dependency declaration there